go-controller/pkg/kubevirt/types.go

```diff
@@ -0,0 +1,5 @@
+package kubevirt
+
+const (
+	AllowPodBridgeNetworkLiveMigrationAnnotation = "kubevirt.io/allow-pod-bridge-network-live-migration"
```

Comment:
Do we really need to have `bridge` in the annotation ? 

Does OVN-Kubernetes care about that detail ?

Reply:
This is a annotation from kubevirt that allows to migrate when a VM has bridge binding at pod network, so nothing we can do about that.